torch/_dynamo/resume_execution.py

```diff
@@ -48,6 +48,107 @@ class ReenterWith:
     stack_index: int
     target_values: Optional[Tuple[Any, ...]] = None
 
+    def try_except_torch_function_mode(self, code_options, cleanup: List[Instruction]):
+        """
+        Codegen based off of:
+        try:
+            (rest)
+        finally:
```

Comment:
It looks like this bytecode is actually doing:
```python
try:
    (rest)
except:
    torch._dynamo.utils.set_torch_function_mode_stack(___prev_torch_function_mode_stack)
```

Update comment to reflect this?